litmus-portal/backend/auth/pkg/server/server.go

```diff
@@ -1,22 +1,36 @@
 package server
 
 import (
+	"context"
+	"log"
 	"net/http"
+	"os"
 	"strings"
 	"time"
 
 	"github.com/dgrijalva/jwt-go"
 	"github.com/gin-gonic/gin"
-	log "github.com/golang/glog"
-
+	glog "github.com/golang/glog"
```

Comment:
you can remove `glog`
```suggestion
	"github.com/golang/glog"
```